backend/src/services/model.ts

```diff
@@ -319,6 +322,18 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   if (modelDiff.settings?.mirror?.destinationModelId && modelDiff.settings?.mirror?.sourceModelId) {
     throw BadReq('You cannot select both mirror settings simultaneously.')
   }
+  if (modelDiff.collaborators) {
+    const invalidUsers = []
+    modelDiff.collaborators.forEach(async (collaborator) => {
+      const userInformation = await authentication.getUserInformation(collaborator.entity)
```

Comment:
I think this should throw an error if the user can't be found. It's a bit difficult as this uses our connectors and the implementation could be anything but hopefully the typing of `getUserInformation()` can enforce that it can't just return nothing